docs/technical-design/database-diagram.md

```diff
@@ -38,5 +40,44 @@ User {
     string role
 }
 
-User }|--|{ State: "many to many"
+Document {
+    string id
+    string name
+    string s3URL
+    string uploadedBy
+    datetime createdAt
+    bool virusScan
+}
+
+User }|--|| Document: "has many"
+
+User }|--|| Question: "has many"
+
+Question }|--|| QuestionResponse: "has many"
```

Comment:
I believe some of these have the relationships backwards.

```suggestion
Question ||--|{ QuestionResponse: "has many"
```

Otherwise looks good to me